interactive_setup.sh

```diff
@@ -231,25 +250,77 @@ if [ "$backend" == 0 ]; then
     fi
 
     (cd "$path" && docker compose up -d)
-    backend=1
-    echo "Backend added"
+    replicationBackend=1
+    echo "replication-backend added"
   elif [ "$app" == 0 ]; then
     curl -X PUT -u "admin:$COUCHDB_PASSWORD" "https://$APP_URL/db/app/_security" -d '{"admins": { "names": [], "roles": [] }, "members": { "names": [], "roles": ["user_app"] } }'
     curl -X PUT -u "admin:$COUCHDB_PASSWORD" "https://$APP_URL/db/app-attachments/_security" -d '{"admins": { "names": [], "roles": [] }, "members": { "names": [], "roles": ["user_app"] } }'
   fi
+
+  if [ "$withReplicationBackend" != "y" ] && [ "$withReplicationBackend" != "Y" ]; then
+    echo "APP_BACKEND_URL=http://couchdb:5984" >> "$path/.env"
+    echo "COMPOSE_PROFILES=" >> "$path/.env"
+  fi
 fi
 
-if [ "$app" == 0 ] && [ "$UPTIMEROBOT_API_KEY" != "" ] && [ "$UPTIMEROBOT_ALERT_ID" != "" ]; then
+if [ "$aamBackendService" == 0 ]; then
   if [ -n "$7" ]; then
-    createsMonitors="$7"
+    withAamBackendService="$7"
+  else
+    echo "Do you want to add aam-backend-services (query-backend)?[y/n]"
+    read -r withAamBackendService
+  fi
+
+  if [ "$withAamBackendService" == "y" ] || [ "$withAamBackendService" == "Y" ]; then
+    if [ "$withReplicationBackend" == "y" ] || [ "$withReplicationBackend" == "Y" ]; then
+      sed -i -e 's/COMPOSE_PROFILES=replication-backend/COMPOSE_PROFILES=replication-backend,aam-backend-service/g' "$path/.env"
+    else
+      sed -i -e 's/COMPOSE_PROFILES=/COMPOSE_PROFILES=aam-backend-service/g' "$path/.env"
+    fi
+
+    echo "AAM_BACKEND_SERVICE_URL=http://aam-backend-service:5984" >> "$path/.env"
+
+    mkdir "$path/config"
+    mkdir "$path/config/aam-backend-service"
+
+    generate_password
+    {
+      echo "CRYPTO_CONFIGURATION_SECRET=$password";
+      echo "SPRING_WEBFLUX_BASE_PATH=/api";
+      echo "SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI=https://keycloak.aam-digital.com/realms/$org";
```

Comment:
can we avoid this direct ref to the prod keycloak server? Maybe we should have a setup.env or get it from the keycloak.json template that is used for the frontend.